src/api.rs

```diff
@@ -118,7 +118,7 @@ pub struct InscriptionRecursive {
   pub charms: Vec<Charm>,
   pub content_type: Option<String>,
   pub content_length: Option<usize>,
-  pub delegate: Option<InscriptionId>,
+  pub delegates: Vec<InscriptionId>,
```

Comment:
We cannot change the return type since it's a recursive endpoint. We'll have to return the effective delegate here instead in an `Option<InscriptionId>`.